runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/AutocompleteWidget/AutocompleteWidget.types.ts

```diff
@@ -1,6 +1,8 @@
 import { WidgetProps } from '@rjsf/core';
 
+import { Options } from '#src-app/hooks/useOptions';
+
 export interface AutocompleteWidgetProps extends WidgetProps {
-    groupBy?: (option: any) => string;
     customErrors?: string[];
+    autocompleteOptions: Options;
```

Comment:
imo its obvious its about the autocomplete, you can remove this part from prop name